plugins/orchestrator-common/src/types.ts

```diff
@@ -65,3 +72,13 @@ export enum WorkflowCategory {
   ASSESSMENT = 'assessment',
   INFRASTRUCTURE = 'infrastructure',
 }
+
+export interface WorkflowOverview {
+  workflowId: string;
+  name?: string;
+  lastTriggered?: string;
```

Comment:
I suggest to rename lastTriggered to lastTriggeredMs so it'll be clear it's milliseconds and it should be an optional number so backend won't need to convert it to string and than UI back to number